cmd/cli/plugin/package/README.md

```diff
@@ -78,35 +97,42 @@ Use "tanzu package repository [command] --help" for more information about a com
 3. Add a repository
 
    ```sh
-   >>> tanzu package repository add standard-repo --url projects-stg.registry.vmware.com/tkg/test-packages/standard-repo:v1.0.0 -n test-ns --create-namespace
-   Added package repository 'standard-repo'
+   >>> tanzu package repository add standard-repo --url projects-stg.registry.vmware.com/tkg/test-packages/standard-repo:v1.0.0 -n test-ns
+   Waiting for package repository to be added
+   4:04:00PM: packagerepository/standard-repo (packaging.carvel.dev/v1alpha1) namespace: test-ns: Reconciling
```

Comment:
is this ```4:04:00PM``` part of the output?

Reply:
Yep.

Reply:
i think in this case it's a good exception to the rule -- we want to provide "non-distracting" indication of time passing as output is being added to the screen, user is not interested in full timestamp on every line as it is just noise.

Reply:
Would be good to document that is a divergence from rest of plugins with the reasoning.

Reply:
Sure. Please let me know the best place to document this.